ideascube/models.py

```diff
@@ -311,6 +311,13 @@ def get_prep_value(self, value):
         return json.dumps(value)
 
 
+class LanguageField(models.CharField):
+    def __init__(self, *args, **kwargs):
+        kwargs['choices'] = get_all_languages()
+
```

Comment:
One thing I really like in python is that those choices are usually not subjective: where to put a space, a blank like, etc. and this make easy for someone to read the code from someone else, because we like at home.
So my opinion is that it's not a good practice to add "subjective" blank lines. So I'd say: we should not add blank lines if not requested by pep8.
Of course, this is all subjective, but… that what I'm trying to say :p